src/main/steps/applicant/statement-of-service/choose-parties/routeGuard.test.ts

```diff
@@ -0,0 +1,79 @@
+import { mockRequest } from '../../../../../test/unit/utils/mockRequest';
+import { mockResponse } from '../../../../../test/unit/utils/mockResponse';
+import { CosApiClient } from '../../../../app/case/CosApiClient';
+
+import { routeGuard } from './routeGuard';
+
+const deleteCitizenStatementDocumentMock = jest.spyOn(CosApiClient.prototype, 'deleteCitizenStatementDocument');
+
+describe('documents > upload > upload-your-documents > routeGuard', () => {
```

Comment:
Do not copy paste test files. The describe is not correct. Validate for all test files.

Reply:
Validated and modified